spyder_notebook/notebookplugin.py

```diff
@@ -174,6 +170,9 @@ def register_plugin(self):
         self.focus_changed.connect(self.main.plugin_focus_changed)
         self.main.add_dockwidget(self)
         self.create_new_client(give_focus=False)
+        img_path = os.path.join(PACKAGE_PATH, 'img_src'+ os.sep +'icon.svg')

```

Comment:
img_path -> icon_path

This can rewritten as

    osp.join(PACKAGE_PATH, 'images', 'icon.svg')